src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -154,14 +155,23 @@
                         this.defaultMessage = "An error occurred fetching Git commits";
                     });
             },
-            getParameterValues(values) {
+            getParameterValues(values, isValid) {
```

Comment:
The next line should possibly be `if (valid)` rather than `if (values)`.

Also: is there a reason why this function doesn't tranform the values directly into the format required for running the report? i.e. `{"key1": "value1", "key2": "value2"}`. This would eliminate the need for further processing [here](https://github.com/vimc/orderly-web/pull/303/files#diff-84983702619418ac4880153048bbd9ddcc05c220414f0a797fc71050d94dbfd0R219).

Reply:
Could do that, though `parameterValues` themselves should be in array form as they get passed into the parameter list component as a prop so would have to stash the transformed params in another value.

Reply:
Ah, I see. Scratch that!